lib/helpers/populate/getModelsMapForPopulate.js

```diff
@@ -390,7 +390,7 @@ function _virtualPopulate(model, docs, options, _virtualRes) {
 
     if (!localField || !foreignField) {
       return new MongooseError('If you are populating a virtual, you must set the ' +
-        'localField and foreignField options');
+        `localField and foreignField options (Model: ${model.modelName})`);
```

Comment:
Nitpick:
Wrapping fields in backticks makes it clear that this an exact identifier that people should be looking for, not just part of the message.
```suggestion
        `\`localField\`, and \`foreignField\` options for model: \`${model.modelName}\``);
```

Reply:
good idea, maybe we should note it down somewhere and change all errors to use this style consistently

Reply:
Thanks 👍